script/check-size

```diff
@@ -106,6 +106,7 @@ markdown()
             markdown_size "$@" >>"${OT_REPORT_FILE}"
             ;;
         post)
+            #cat "${OT_REPORT_FILE}"
```

Comment:
what's this for?

Reply:
Ah! Thanks. 
It's left-over from debugging script. Will remove it.